src/Model/LogEntry.php

```diff
@@ -18,18 +20,23 @@ class LogEntry extends DataObject
     private static $table_name = 'LogEntry';
 
     /**
+     * @var bool
+     */
+    private static $format_entry = true;
+
+    /**
      * {@inheritDoc}
      */
     private static $db = [
-        'Entry'    => 'Text',
-        'Level'    => 'Varchar'
+        'Entry' => 'Text',
+        'Level' => 'Varchar'
     ];
 
     /**
      * {@inheritDoc}
      */
     private static $summary_fields = [
-        'Entry',
+        'Entry' => 'GridfieldSummary',
         'Created',
         'Level'
```

Comment:
This might be causing the tests to fail. Maybe need to re-define the other keys as values too?

Also, can you add some tests for the new logic?